src/SFA.DAS.ApprenticeCommitments/Application/Commands/CreateApprenticeship/CreateApprenticeshipCommandHandler.cs

```diff
@@ -40,7 +40,8 @@ public class CreateApprenticeshipCommandHandler : IRequestHandler<CreateApprenti
                 ApprenticeshipId = command.ApprenticeshipId,
                 Email = command.Email,
                 EmployerName = command.EmployerName,
-                EmployerAccountLegalEntityId = command.EmployerAccountLegalEntityId
+                EmployerAccountLegalEntityId = command.EmployerAccountLegalEntityId,
+                TrainingProviderName = "Provisional Training Provider Name",
             });
```

Comment:
I'm wondering where are we getting the TrainingProviderId from. We actually have a choice of getting it from the event or from the call to commitments.

Reply:
That will be part of CS-286